uiowa/uiowa
# Thank You Form Checkbox

Resolves #6251 

# How to test

1. `blt ds --site=default.uiowa.edu`
2. Enable thank you form: https://default.prod.drupal.uiowa.edu/admin/config/sitenow/sitenow-dispatch/thanks
3. Add 
```
$config['sitenow_dispatch.settings']['thanks']['api_key'] = 'xxx';
$config['sitenow_dispatch.settings']['thanks']['hr_token'] = 'xxx'; 
```
to `docroot/sites/default/settings/local.settings.php` and DM Makur for the `api_key` and `hr_token`
4. Place thank you form block on a page.
5. Check and uncheck various settings (supervisor email, additional email, etc.) https://default.uiowa.ddev.site/admin/config/sitenow/sitenow-dispatch/thanks and test form submissions.
6. If "`Please feel free to include this thank-you in staff recognition events or materials.`" is checked on the thank you form, this text "`This message has been approved to be included in staff recognition events or materials.`" should show up in the email footer.
